docs/data/joy/components/input/input.md

```diff
@@ -166,6 +166,24 @@ You can use these to customize the component with both the `sx` prop and the the
 
 {{"demo": "InputVariables.js", "hideToolbar": true}}
 
+## Integration with 3rd party libraries
+
+To integrate Input with 3rd party libraries, create a wrapper component to map props with 3rd party component API.
+
+Then use the wrapper component as a value to `slotProps.input.component` of Joy UI Input.
```

Comment:
```suggestion
Then, use the wrapper component as a value to the `slotProps.input.component` slot of the Joy UI Input.
```

Is slot the correct name for what `slotProps.input.component` is?

Reply:
In my opinion, the slot is the input but `slotProps.input.component` is the specific property that will be passed to input slot.

Reply:
Gotcha, so maybe this is more accurate?

```suggestion
Then, use the wrapper component as a value to the `slotProps.input.component` property of the Joy UI Input.
```